packages/contracts/contracts/ChugSplashManager.sol

```diff
@@ -116,12 +119,14 @@ contract ChugSplashManager is Owned {
     /**
      * @notice Address of the ChugSplashRegistry.
      */
-    ChugSplashRegistry public immutable registry;
+    // TODO: Add immutable keyword once this contract is not upgradeable anymore.
```

Comment:
I think we can get rid of the TODOs here, we should just a thorough review when we transition away from upgradeability.